lobby-db/launch_db

```diff
@@ -3,12 +3,40 @@
 # Convenience script to get a local DB up and running on docker
 # with some sample data.
 
-set -eu
+set -u
+set -e
```

Comment:
Why this change?
Shouldn't it be the exact same?

Reply:
Left over from debugging, should be converted back to how it was.

Reply:
Fixed: cb976f3d7